### ISSUE TITLE
Parse shutdown. What’s the future of the BaaS?

### ISSUE THREAD

**Issue by sgrove** [sgrove/rfcrf-tanmaioneblog#5]
---

Today, Parse [anounced](http://blog.parse.com/announcements/moving-on/) that they are winding down the Parse service. It’ll be completely retired in an year. What Parse started was amazing: making it super fast to set up applications on the web and mobile. But with it shutting down, the developers who built apps on Parse are now faced with two difficult choices:

1.  Migrate to another BaaS (and risk this whole episode again).
1.  Run their own backend (comes with its downsides).

Developers will now be skeptical about using any BaaS. It can be acquired and shut down. They have every reason to be. [Hasura](http://hasura.io/) is a platform for building scalable backends. It is however built around this core philosophy:

> ### Developers should have complete control over their backend

With [Hasura](https://hasura.io/), every app gets its own backend, runs on its own dedicated server (or a cluster) unlike a monolithic service that serves every app (like Parse, Firebase etc). This gives you as a developer, a unique ability to completely customise your backend, but _only if you have to_.

In short, we are as dev friendly as Parse and as customisable as GCE or AWS. So, there will never be a question of moving away from [Hasura](https://hasura.io/); you are not locked in to begin with.

We would like to share a lot more about the tech behind [Hasura](https://hasura.io/). We will be making a beta release announcement early next week. Stay tuned.

_This post originally appeared in our blog: <http://hasura.io/blog/our-philosophy-as-a-baas/>. Please sign up for early access at Hasura.io._

```backmatter
{"source":"medium","postId":"dee79fec494d","publishedDate":1454058144613,"url":"https://medium.com/@HasuraHQ/parse-shutdown-what-s-the-future-of-the-baas-dee79fec494d"}
```
